Add initial book page for Enterprise Contract

simonbaird opened the pull request:
Ref: https://issues.redhat.com/browse/HACBS-1374

lcarva reviewed (commented):
Good progress so far!

simonbaird commented on book/enterprise-contract.md:
```diff
@@ -0,0 +1,85 @@
+## Tekton Chains
+
+Even though Enterprise Contract depends on Tekton Chains to create and sign
+container image attestations, and the Enterprise Contract team members were
```
Agreed. The teams and responsibilities are not on topic here.

simonbaird commented on book/enterprise-contract.md:
```diff
@@ -0,0 +1,85 @@
+
+todo
+
+### Enterprise Contract Renovate Bot
```
Done.

simonbaird commented on book/enterprise-contract.md:
```diff
@@ -0,0 +1,85 @@
+Built-in requirements are reasonably static and include the following:
+
+- The container image is signed with a known and trusted signing secret
+- The container image has a pipeline run attestation created by Tekton Chains
```
Fixed.

simonbaird commented on book/enterprise-contract.md:
```diff
@@ -0,0 +1,159 @@
+
+It's expected that a GitHub Action, or similar automation would automatically
+produce a pull request with the required update to the list when a new trusted
+task bundle is created.
```
@lcarva I'm not sure I explained this well since I'm a little fuzzy on how it works. Lmk if you have some ideas how to describe this better.

simonbaird replied:
Could also be something to think about and iterate on later...

robnester-rh commented on book/enterprise-contract.md:
```diff
@@ -0,0 +1,159 @@
+
+The reference set of rego policies created for the HACBS MVP are defined
+[here](https://github.com/hacbs-contract/ec-policies/) and are documented
+[here](https://hacbs-contract.github.io/ec-policies/). In includes rules for a
```
Typo? Should this be `It includes`

amfred commented:
We might just need two approvals?  I added one.

gorkem commented on book/enterprise-contract.md:
```diff
@@ -0,0 +1,171 @@
+
+The task is defined
+[here](https://github.com/hacbs-contract/ec-cli/blob/main/deployments/verify-enterprise-contract.yaml).
+
```
the link is broken

gorkem commented on book/enterprise-contract.md:
```diff
@@ -0,0 +1,171 @@
+The ECP CRD defines a Kubernetes CR which is used to hold the configuration
+needed for running a specific instance of the Enterprise Contract. This
+includes the public key required to verifying signatures, the list of policy
+and data sources, and any other required configuration.
```
verifying -> verify

gorkem commented on book/enterprise-contract.md:
```diff
@@ -0,0 +1,171 @@
+
+### EC Policies
+
+The reference set of policy rules created for the HACBS MVP are defined
```
I suppose this is going to be the location for default/reference policies not only for MVP and afterwards too. Let's remove the qualifier "HACBS MVP" only.

gorkem commented on book/enterprise-contract.md:
```diff
@@ -0,0 +1,171 @@
+To verify that tasks used in the build pipeline are from known and trusted
+Tekton bundles, EC requires a list of those bundles.
+
+HACBS users can leverage the Renovate Bot service to keep such Tekton bundle
```
Link to Renovate

gorkem reviewed (changes requested):
I have a few small cosmetic changes only

simonbaird commented:
Thanks very much for the thoughtful review @gorkem. Addressed your comments in [this revision](https://github.com/redhat-appstudio/book/compare/bf1fd6551a2dd52109f3bed9e41591036db06242..403be2650891b4e5e57ab8fd560cfa0ab4c7eed3).

simonbaird commented:
In f3b5dc1I just capitalized "Conftest" in two places.

gorkem reviewed (approved):
Thanks

simonbaird commented:
Safe to merge?
